src/Common/src/Interop/Windows/mincore/Interop.OpenCommPort.cs

```diff
@@ -9,9 +9,9 @@
 
 internal partial class Interop
 {
-    internal partial class KernelBase
+    internal partial class mincore
     {
-        [DllImport(Libraries.KernelBase, SetLastError = true)]
+        [DllImport(Libraries.CoreComm_L1_1_1, SetLastError = true)]
```

Comment:
This will not work on older Windows versions. Why are we doing this change?

Reply:
This change was requested by Windows team. The API is only used by UWP and was introduced in RS3, for Windows non-Uap it uses kernel32!CreateFile.

Reply:
The Interop.OpenCommPort.cs  file should be compiled in for Uap only then.

Reply:
It is via the wrapper in System.IO.Ports and the settings in the project:
https://github.com/dotnet/corefx/blob/c3e50ed1c017b36c5b8fe19a5dc9c499369eeb43/src/System.IO.Ports/src/System/IO/Ports/SerialStream.Uap.cs#L13
https://github.com/dotnet/corefx/blob/a989d4f91cf4208b7a2481d120498a581328eb1d/src/System.IO.Ports/src/System.IO.Ports.csproj#L139

Reply:
Could you please move `$(CommonPath)\Interop\Windows\mincore\Interop.OpenCommPort.cs` under this condition as well?

Reply:
Yes, that makes things clear and explicit, added.